tests/forte/data/data_store_test.py

```diff
@@ -596,6 +636,26 @@ def test_add_group_raw(self):
             ["test_group", [], tid, "forte.data.ontology.top.Group"],
         )
 
+    # def test_add_link_raw(self):
```

Comment:
if we remove these test cases, do we have test cases for `add`?

Reply:
They are added from other PRs.